vite.config.js

```diff
@@ -15,6 +16,18 @@ export default ({ mode }) => {
       viteTsconfigPaths(),
       svgrPlugin(),
       { ...polyfillNode({ fs: true }), enforce: "post" },
+      transformPlugin({
+        tStart: "%{",
+        tEnd: "}%",
+        replace: {
+          "COOLER_LOANS_API_ENDPOINT":
+            process.env.VITE_COOLER_LOANS_API_ENDPOINT ?
+              process.env.VITE_COOLER_LOANS_API_ENDPOINT :
```

Comment:
shouldn't this be import.meta.env?

Reply:
This works, so I assume that vite supports `process.env` too. Would you prefer that I use `import.meta.env`?

Reply:
no i guess i would leave it then... that's weird

Reply:
maybe cause it's the config https://main.vitejs.dev/config/#using-environment-variables-in-config